vital/types/bounding_box.h

```diff
@@ -95,6 +95,33 @@ class bounding_box
     m_bbox =  Eigen::AlignedBox< T, 2 >( ul, lr );
   }
 
+  /**
+   * @brief Create default (invalid) box.
+   */
+  bounding_box()
+  {
+    // Note That any initial state logic here
```

Comment:
Pedantic: your capitalization (and maybe where you put the line break) is odd. Also, consider: `// NOTE: Any initial ...`; highlighters will tend to mark this differently. (`CAUTION` and `WARNING` work also, though not sure something stronger than `NOTE` is needed.)